storage/series/series_test.go

```diff
@@ -531,3 +531,32 @@ func TestSeriesOutOfOrderWritesAndRotate(t *testing.T) {
 	require.NoError(t, it.Err())
 	require.Equal(t, expected, actual)
 }
+
+func TestSeriesWriteReadFromTheSameBucket(t *testing.T) {
+	opts := newSeriesTestOptions()
+	opts = opts.SetRetentionOptions(opts.RetentionOptions().
+		SetRetentionPeriod(40 * 24 * time.Hour).
+		SetBlockSize(5 * 24 * time.Hour).
```

Comment:
Could you mention in a comment above the test that this uses a block size (5d) that is not equally as divisible as seconds from time zero and time epoch and tests this compatibility with reads?

Reply:
Sure